lab1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="app.css">
+	<title>Homepage</title>
+</head>
+<body>
+		<header><h1>{Nightbringers}</h1></header>
+		<div class="vertical-menu">
+				<a href="#" class="active">Kings of the NightWorld, where they lurk.</a>
+				<a href="#">What a horrible night to have a curse. Fight the undead through the perpetual night.</a>
+				<a href="#">{The looming night falls upon the meek and frail turning them into horrible abominations.
+					Creatures from folklore, they hunt you down.</a>
+				<a href="#">{More story to come ...}</a>
+				<a href="#">{More story to come ...}</a>
+				
+			</div>
+	<main>
+		<button class="but">{Action}</button>
+		<p><h2>Skulls: 100</h2></p>
```

Comment:
Suggestion: Maybe wrap button and Skulls together in a `div` so you can get better height expectation
![screen shot 2018-02-11 at 6 12 28 pm](https://user-images.githubusercontent.com/2566047/36081690-27c14aae-0f57-11e8-8e2a-74fa7aca9e72.png)

Also you can give better margin of the whole row